Jasmine/Common/Models/Level.swift

```diff
@@ -0,0 +1,50 @@
+import RealmSwift
+
+class Level: Object {
+
+    dynamic var name: String = "Untitled Level"
+    // difficulty of the level, higher means more difficult
+    // 1 since realm doesn't allow non-default values
+    dynamic var difficulty: Int = 1
```

Comment:
Do we still have `difficulty`...?

Also, you're not storing the phrases in this object?